content/authors/Valentina-Borghesani/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+# Display name
+title: "Valentina Borghesani"
+
+# Username (this should match the folder name and the name on publications)
+authors:
+  - "Valentina Borghesani"
+github: vborghesani
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence) & Organizations/Affiliations
+position: "Post doctoral fellow at Université de Montréal, Montréal, Québec, Canada"
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+  - icon: envelope
+    icon_pack: fas
+    link: "mailto:valentinaborghesani@gmail.com" # For a direct email link, use "mailto:test@example.org".
+  - icon: twitter
+    icon_pack: fab
+    link: https://twitter.com/vborghesani
+  - icon: github
+    icon_pack: fab
+    link: https://github.com/vborghe
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+  - Brainhack Global Organizing Committee
+# - Volunteers
```

Comment:
```suggestion
  - Logistics
  - Outreach
  - DEI
```

Reply:
According to #41

Reply:
I'd fine if all these were edited to match #41 in your upcoming commits. I honestly did not pay attention to this field.